pkg/api/types.go

```diff
@@ -2263,3 +2263,83 @@ type ThirdPartyResourceData struct {
 
 	Data []byte `json:"name,omitempty"`
 }
+
+// PodSecurityPolicy governs the ability to make requests that affect the SecurityContext
+// that will be applied to a container.
+type PodSecurityPolicy struct {
+	TypeMeta   `json:",inline"`
+	ObjectMeta `json:"metadata,omitempty"`
+
+	// Privileged determines if a container can request to be run as privileged.
+	Privileged bool `json:"privileged,omitempty"`
+	// Capabilities is a list of capabilities that can be requested to add to the container.
+	Capabilities []Capability `json:"capabilities,omitempty"`
+	// HostPath determines if the policy allows containers to use the HostDir volume plugin
+	HostPath bool `json:"hostPath,omitempty"`
```

Comment:
Do we want to diffrentiate between RW and RO? What about allowing only certain paths, e.g. only allowing /etc/ssl to be mounted RO?